test/e2e/storage/drivers/csi.go

```diff
@@ -644,16 +644,9 @@ func WaitForCSIDriverRegistrationOnNode(nodeName string, driverName string, cs c
 	return nil
 }
 
-func deleteNamespaceFunc(cs clientset.Interface, ns string, timeout time.Duration) func() {
+func deleteNamespaceFunc(f *framework.Framework, namespace string) func() {
```

Comment:
Do we need this function? Since the deletion part of f.DeleteNamespace(namespace) already running in another go routine

Reply:
A nice idea.

Reply:
fixed